docker-compose.yaml

```diff
@@ -160,10 +162,8 @@ services:
     ports:
       - 9001:9001
       - 4001:4000
-    depends_on:
-      - server-pulsar
```

Comment:
As above, I've removed this because it was removed from `executor-legacy` in #3047.